[2022-05-26 21:00:51,878] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_1 scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-05-26 21:00:52,058] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_1 scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-05-26 21:00:52,065] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 21:00:52,068] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-26 21:00:52,076] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 21:00:52,562] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): task_1> on 2022-05-24 00:00:00+00:00
[2022-05-26 21:00:52,644] {standard_task_runner.py:52} INFO - Started process 396 to run task
[2022-05-26 21:00:52,753] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'trigger_rule', 'task_1', 'scheduled__2022-05-24T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpy2liv9y8', '--error-file', '/tmp/tmpkudfo5j9']
[2022-05-26 21:00:52,819] {standard_task_runner.py:80} INFO - Job 11: Subtask task_1
[2022-05-26 21:00:53,762] {task_command.py:369} INFO - Running <TaskInstance: trigger_rule.task_1 scheduled__2022-05-24T00:00:00+00:00 [running]> on host 2fe522ea8532
[2022-05-26 21:00:55,495] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-24T00:00:00+00:00
[2022-05-26 21:00:55,507] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-26 21:00:55,509] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'exit 0']
[2022-05-26 21:00:55,891] {subprocess.py:85} INFO - Output:
[2022-05-26 21:00:56,057] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-26 21:00:56,596] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_1, execution_date=20220524T000000, start_date=20220526T210051, end_date=20220526T210056
[2022-05-26 21:00:59,532] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-26 21:01:00,878] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
